packages/test-base/src/commonTest/kotlin/io/realm/kotlin/test/common/notifications/RealmNotificationsTests.kt

```diff
@@ -90,6 +90,45 @@ class RealmNotificationsTests : FlowableTests {
         }
     }
 
+    @Test
+    fun registerTwoFlows() = runBlocking {
```

Comment:
This was covered in `cancelAsFlow`, but I guess it is nice with an explicit test.